statchatui/src/SignIn/SignIn.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./SignIn.css";
+
+const SignIn = () => {
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const formData = {
+      Username: event.target.username.value,
+      Password: event.target.password.value,
+    };
+
+    try {
+      const response = await fetch("http://localhost:3002/users/login", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(formData),
+      });
+
+      const data = await response.json();
+      if (response.ok) {
+        alert("login succesful");
+      } else {
+        alert(`Login failed: ${data.message}`);
+      }
+    } catch (error) {
+      console.error("Error submitting form:", error);
+    }
+  };
+  return (
+    <div className="container">
```

Comment:
Remove these `className` references if the css file doesn't exist

Reply:
i am going to add css for it later, i will commit the css files but they are blank as of right now